book/website/project-design/pd-checklist.md

```diff
@@ -0,0 +1,69 @@
+(pd-checklist)=
+# Getting Started Checklist 
+
+We can begin the project design process by identifying different parts of our research, such as main research questions, methods and materials, code and data requirements, workflow, communication channels, ways of working, collaborative practices, and so on. 
+This process allows us to be intentional from the start to ensure that our research is reproducible, well-communicated, and inclusive of all stakeholders where decisions are collaboratively made. 
+We can explore and select the right tools and methods for reproducibility in our research and promote good practices such as documentation, version control, peer-review processes, testing, workflow, archiving, and data management plans from the beginning. 
+Finally, we can plan for publishing and sharing research components before, during, and after the project. 
+
+**Below is a checklist you can use to help identify areas of project planning you might want to look at.**
+
+### Aims & Values 
+* Define the main research questions and objectives.
+* Identify the core values and principles that guide your project.
+* Useful documentation: [project canvas](https://canvanizer.com/new/project-canvas), values document, [project 1-pagers](https://www.smartsheet.com/content/project-report-templates).
+
+### Timeline & Milestones 
+* Establish a project timeline with key milestones, output, and deadlines.
+* Break down the project into manageable phases or tasks.
+* Useful documentation: [Gantt charts](https://clickup.com/blog/gantt-chart-project-templates/), [roadmaps](https://www.smartsheet.com/free-product-roadmap-templates-smartsheet), or [project boards](https://teamhood.com/project-management/project-board/).
+
+### Methodology
+* Determine the appropriate research methods and materials.
+* Consider the necessary code and data requirements for your project.
+* Document the workflow for data collection, analysis, and interpretation.
+* Define data governance processes to ensure data is kept securely, used appropriately, and complies with data regulations in relevant countries/geographic areas. 
+* Useful documentation: [data management plan](rr-rdm-dmp), [code repository](pd-project-repo), [electronic lab notebooks](rr-open-notebooks), [Data Governance Chapter](pd-dg)
+
+### Operations 
```

Comment:
I am developing a section on project management frameworks, and I wonder if/where that would fit with this checklist? Not something that needs immediate resolution, but something that I'm keeping in mind here. See #3386.

Reply:
This is intended to be an overview checklist so we could totally update this with references to your PR either as part of this week or as a follow up PR for this file once it's merged - I was thinking it would either go in Operations or as part of the methodology sections?

Reply:
Agree to leave as is rn, then update when we introduce content on framworks